components/n-ui/tracking/third-party/floodlight.js

```diff
@@ -19,6 +19,9 @@ module.exports = function (flags) {
 	const isSignUpForm = /^\/signup/.test(location.pathname);
 	const isSubscriptionConfirmation = /^\/thank-you/.test(location.pathname);
 	const isTrialConfirmation = (l) ? l.getAttribute('data-signup-is-trial') === 'true' : undefined;
+	const isBarrier = !!document.querySelector('[data-trackable="page:product"]');
+	const isSubscriber = self.FT.flags.subscriberCohort;
```

Comment:
Where is self defined?

Reply:
https://stackoverflow.com/questions/3638887/whats-the-difference-between-self-and-window 😉

Reply:
![mindblown](https://user-images.githubusercontent.com/1721150/50978021-64a64880-14eb-11e9-9ea1-af7fcfa7a264.gif)

I would say `window` is more well known and readable? `self` has been misused as scoped `this` for far too long.